src/objects/Hierarchy_Settings__c.object

```diff
@@ -239,6 +239,28 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
```

Comment:
If you do a full metadata retrieve of the metadata from the org (do it when you don't have any pending work to commit!), there should be updates to our translation files for these new fields.